### PR TITLE
Headcrabs can be spawned in the infestation event

### PR DESCRIPTION
**Pull request by gurfan**
what it says on the tin 
[tweak]

:cl:
 * rscadd: Headcrabs can now be spawned in the infestation event.

### CODE REVIEW

File: __DEFINES/setup.dm [hunk 1]
```diff
@@ -758,6 +758,7 @@ SEE_PIXELS	256
 #define VERM_MUSHMEN 15
 #define VERM_FROGS 14
 #define VERM_SNAILS 15
+#define VERM_HEADCRABS
```
Comment (boy2mantwicethefam) [inline __DEFINES/setup.dm]: add a number here

Comment (gurfan) [reply __DEFINES/setup.dm]: oops

**Comment by DrSnips** [comment]
Here's my issue with headcrabs. They are basically "self-antag"ing tools because the result in player zombies if you "accidentally" get one and player zombies are no joke.
Personally I don't think they are a good idea even if they are a limited time self-antag tool because most likely you will be revived (or at least someone will attempt to) and there's no real reason you shouldn't try for it until AFTER being antag enabled.

**Comment by gurfan** [comment]
Considering that headcrabs zombies have a chance to get brained after dying I dont think the brief time as a zombie will be worth it. Zombies are already not too fun considering that you can open access airlocks or firelocks or anything. I think that they should be tested and if people throwing themselves into headcrabs is a legitimate problem then the zombies can be made into mindless ones anyway.

**Comment by DrSnips** [comment]
Im pretty sure they can do the "you hear the motors of the airlock whirr" opening thing. Either way I'm putting my money down on players getting free valids. Go for it but don't say I didn't tell you so.

**Pull request merged**